data/remnant/remnant jobs.txt

```diff
@@ -223,6 +224,115 @@ mission "Remnant: Bounty 5"
 
 
 
+mission "Remnant: Logistics Civil Engineering"
+	name "Civil delivery to <planet>"
+	description `Engineers on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant civil" 30 80
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
+		has "Remnant: Logistics 1: done"
+		random < 51
```

Comment:
Why all the odd numbers?

Reply:
I was using less than so to be 50/50 I needed < 51. I could have use <=. Dealer's choice.